odata-url-conventions/4 Resource Path.md

```diff
@@ -998,4 +1000,39 @@ Content-Type: text/plain
 
 $filter=[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]
 ```
+This POST request would result from submitting the HTML form
+```html
+<form method="post" action="http://host/service/People/$query"
+      enctype="text/plain">
+  <input name="$filter"
+    value='[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]'>
+</form>
+```
+:::
+
+For content-type `application/x-www-form-urlencoded`, the individual query options MUST be separated by `&`
+and their names and values MAY be percent-encoded even for characters other than `%26` (ampersand) and
+`%3D` (equals).
```

Comment:
Isn't that defined by `application/x-www-form-urlencoded`?

We need a normative reference and point to it here.